src/table/native/Table.jsx

```diff
@@ -105,6 +118,9 @@ const Table = ({ layout, model, manageData, selectionsAPI, changeSortOrder, app,
         onColumnsResized={onColumnsResized}
         clearSelections={clearSelections}
         onHeaderPressed={onHeaderPressed}
+        freezeFirstColumn={freezeFirstColumn(layout)}
```

Comment:
Easier to just do
```suggestion
        freezeFirstColumn={freezeFirstColumn(layout.scrolling?.keepFirstColumnInView)}
```

than to create a file that you have to type the props for?